### PR TITLE
Remove old scss code and relocate rules to more appropriate files

### LINKED ISSUE #6499: [SCSS] Remove old SCSS code
I spotted some SCSS code that was last updated 4 years ago and is possibly obsolete by now. Remove the code if it is no longer in use.

Examples:

- https://github.com/mozilla/foundation.mozilla.org/blob/master/source/sass/views/contact.scss
- https://github.com/mozilla/foundation.mozilla.org/blob/master/source/sass/views/about.scss
- etc

### PR DESCRIPTION
**Pull request by mmmavis**
Closes #6499

### CODE REVIEW

File: source/sass/views/about.scss [hunk 1]
```diff
@@ -1,3 +0,0 @@
-.about-graphic {
```
Comment (mmmavis) [inline source/sass/views/about.scss:1]: About page no longer looks like https://github.com/mozilla/foundation.mozilla.org/issues/153#issue-214165790

File: source/sass/views/contact.scss [hunk 2]
```diff
@@ -1,5 +0,0 @@
-.hear-from-you-line-art {
-  background: url(../_images/line-art/love-to-hear-from-you.svg) no-repeat
-    center 0;
-  min-height: 9rem;
-}
```
Comment (mmmavis) [inline source/sass/views/contact.scss:5]: About page no longer looks like https://github.com/mozilla/foundation.mozilla.org/issues/153#issue-214165790

File: source/sass/views/initiatives.scss [hunk 3]
```diff
@@ -1,7 +0,0 @@
-.bg-white {
```
Comment (mmmavis) [inline source/sass/views/initiatives.scss:1]: Ruleset has been relocated to `source/sass/utilities.scss`

File: source/sass/views/initiatives.scss [hunk 4]
```diff
@@ -1,7 +0,0 @@
...
-  background: $white;
-}
-
-.bottom-line {
```
Comment (mmmavis) [inline source/sass/views/initiatives.scss:5]: Ruleset has been relocated to `source/sass/views/participate.scss` as participate page is the only page that uses this class.

File: source/sass/views/projects.scss [hunk 5]
```diff
@@ -1,15 +0,0 @@
...
-
-.projects-row {
-  border-bottom: 1px solid $gray-05;
-}
-
-#view-projects .btn {
-  white-space: initial;
-}
-
-a.pulse-project {
```
Comment (mmmavis) [inline source/sass/views/projects.scss:13]: We no longer have a [Project page/view](https://github.com/mozilla/foundation.mozilla.org/issues/90#issuecomment-289945507) however these rules are still being used on pulse project list so I've
relocated this chunk to `source/js/components/pulse-project-list/pulse-project-list.scss`

File: source/js/components/pulse-project-list/pulse-project-list.scss [hunk 6]
```diff
@@ -20,10 +20,13 @@
 
   .project-image {
     width: 100%;
-    max-height: none;
```
Comment (mmmavis) [inline source/js/components/pulse-project-list/pulse-project-list.scss:23]: This was here to override the [now-obsolete rule](https://github.com/mozilla/foundation.mozilla.org/pull/6514/files#diff-9831057300166c3b2df655ac821251dde9350c4cc319679818cd76bf711e5244L1-L3) 
```scss
.project-image {
  max-height: 11em;
}
```

**Review by Pomax — APPROVED** [review]

**Pull request merged**